playground/src/pages/postprocessing/scanline.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { ContactShadows, Environment, OrbitControls } from '@tresjs/cientos'
+import { TresCanvas } from '@tresjs/core'
+import { TresLeches, useControls } from '@tresjs/leches'
+import { NoToneMapping, Vector2 } from 'three'
+import { watchEffect } from 'vue'
+import { BlendFunction } from 'postprocessing'
+import { EffectComposerPmndrs, ScanlinePmndrs } from '@tresjs/post-processing'
+
+import '@tresjs/leches/styles'
+
+const gl = {
+  clearColor: '#ffffff',
```

Comment:
Non-blocking, effect looks better with darker bg @damienmontastier 

```suggestion
  clearColor: '#4f4f4f',
```